src/redux/Categories/categories.js

```diff
@@ -0,0 +1,35 @@
+// Action Types
+export const SHOW_STATUS = 'Books/books/SHOW_STATUS';
+
+// initial State
+const initialState = {
+  categories: [],
+  isLoading: false,
+};
+
+// Reducers
+const categoriesReducer = (state = initialState, action) => {
+  const { type, payload } = action;
+
+  switch (type) {
+    case SHOW_STATUS:
+      return {
+        ...state,
+        categories: payload,
+        isLoading: false,
+      };
+
+    default:
+      return initialState;
+  }
+};
+
+// Actions
+export const checkStatus = () => (dispatch) => {
+  dispatch({
+    type: SHOW_STATUS,
+    payload: ['Under Construction'],
+  });
+};
+
+export default categoriesReducer;
```

Comment:
- [ ] You're required to use [redux toolkit](https://redux-toolkit.js.org/) to add action to your app. Kindly follow the redux toolkit way of adding actions.